README.md

```diff
@@ -1,49 +1,71 @@
+<p align="center">
+<a href="https://neo.org/">
+      <img
+      src="https://neo3.azureedge.net/images/logo%20files-dark.svg"
+      width="250px" alt="neo-logo">
+  </a>
+</p>
+
+<p align="center">      
+  <a href="https://travis-ci.org/neo-project/neo-modules">
+    <img src="https://travis-ci.org/neo-project/neo-modules.svg?branch=master" alt="Current TravisCI build status.">
```

Comment:
Perhaps we can leave this comment for now, I think that status is not enabled for `master` branch, correct @erikzhang?